### PR TITLE
Update helloapache to also have a docker provider

### PR DESCRIPTION
**Pull request by bexelbie**
Updates to README as well

### CODE REVIEW

File: examples/helloapache/README.md [hunk 1]
```diff
@@ -2,16 +2,18 @@ This is an atomic application based on the nulecule specification. Kubernetes is
...
 [sudo] atomic run projectatomic/helloapache
 ```
 
-## Option 2: unattended
+## Option 2: Unattended
 
-1. Create file `answers.conf` with these contents:
+1. Create the file `answers.conf` with these contents:
+
+    This sets up the values for the two configurable paramaters (image and hostpost) and indicates that kubernetes should be the orchestration provider.
```
Comment (vpavlin) [inline examples/helloapache/README.md]: typo - hostport

File: examples/helloapache/README.md [hunk 2]
```diff
@@ -38,14 +42,16 @@ You may want to download the application, review, edit the answerfile then run.
...
-1. Edit `answers.conf`, review files if desired and run
+1. Edit `answers.conf`, review files if desired and then run
 
         $ [sudo] atomic run projectatomic/helloapache
         ...
         helloapache
 
 ## Test
-Any of these approaches should create a kubernetes pod. Once its state is "Running" curl the minion it's running on.
+Any of these approaches should create a kubernetes pod or a running docker image.
```
Comment (vpavlin) [inline examples/helloapache/README.md]: s/image/container/ - ...a running docker container

File: examples/helloapache/README.md [hunk 3]
```diff
@@ -38,14 +42,16 @@ You may want to download the application, review, edit the answerfile then run.
...
 
         $ [sudo] atomic run projectatomic/helloapache
         ...
         helloapache
 
 ## Test
-Any of these approaches should create a kubernetes pod. Once its state is "Running" curl the minion it's running on.
+Any of these approaches should create a kubernetes pod or a running docker image. 
+
+With a kuberntes pod, once its state is "Running" curl the minion it's running on.
```
Comment (vpavlin) [inline examples/helloapache/README.md]: typo - kubernetes

**Comment by vpavlin** [comment]
LGTM when the typos are fixed. Thanks @bexelbie!

**Comment by bexelbie** [comment]
Typos to test @vpavlin should be fixed :P

**Pull request merged**